fontforgeexe/cvpalettes.c

```diff
@@ -1605,30 +1650,28 @@ static void CVLayers2Set(CharView *cv) {
 	layer2.layers[i+1] = BDFCharFromLayer(cv->b.sc,i);
     layer2.active = CVLayer(&cv->b)+1;
 
-    GScrollBarSetBounds(GWidgetGetControl(cvlayers2,CID_SB),0,cv->b.sc->layer_cnt+1-2,
-	    CV_LAYERS2_VISLAYERS);
-    if ( layer2.offtop>cv->b.sc->layer_cnt-1-CV_LAYERS2_VISLAYERS )
-	layer2.offtop = cv->b.sc->layer_cnt-1-CV_LAYERS2_VISLAYERS;
-    if ( layer2.offtop<0 ) layer2.offtop = 0;
-    GScrollBarSetPos(GWidgetGetControl(cvlayers2,CID_SB),layer2.offtop);
-
-    GDrawRequestExpose(cvlayers2,NULL,false);
+    CVLayers2Reflow(cv);
 }
 
 static void Layers2Expose(CharView *cv,GWindow pixmap,GEvent *event) {
     int i, ll;
     const char *str;
-    GRect r;
+    GRect r, oldclip;
     struct _GImage base;
     GImage gi;
     int as = (24*cv->b.sc->parent->ascent)/(cv->b.sc->parent->ascent+cv->b.sc->parent->descent);
-
-    if ( event->u.expose.rect.y+event->u.expose.rect.height<CV_LAYERS2_HEADER_HEIGHT )
-return;
```

Comment:
Oops, that shouldn't have been deleted. I guess it's trying to optimise since if it's in the header area, that's already drawn by the ggadget. Anyway, I'll add it back.